config/routes.rb

```diff
@@ -9,7 +9,7 @@
 
     resources :pages, as: :site_pages, controller: 'sites/pages', path: 'p', only: [:show]
     get :pages, to: 'sites/pages#index', as: :site_pages
-    #old routes
+   #old routes
```

Comment:
Incorrect indentation detected (column 3 instead of 4).<br>Missing space after #.